runtime/caml/custom.h

```diff
@@ -48,23 +48,24 @@ struct custom_operations {
 #define custom_compare_ext_default NULL
 #define custom_fixed_length_default NULL
 
-#define Custom_ops_val(v) (*((struct custom_operations **) (v)))
+#define Custom_ops_val(v) (*((const struct custom_operations **) (v)))
```

Comment:
OK for this "constification", even though it is independent from the other Multicore changes.